src/homepage/Components/AboutUs.js

```diff
@@ -3,9 +3,13 @@ import { Jumbotron, Container, Image, Row, Col } from "react-bootstrap";
 import styles from "../Css/AboutUs.module.css";
 import AboutUsImg from "../assets/AboutUsImg.png";
 
-export default function AboutUs() {
+export default function AboutUs({ id }) {
   return (
-    <Jumbotron style={{ paddingTop: "0px", backgroundColor: "#ffffff" }} fluid>
+    <Jumbotron
+      id="about"
+      style={{ paddingTop: "0px", backgroundColor: "#ffffff" }}
```

Comment:
If possible, please avoid inline styling. It creates an issue when we scale up our system.